src/containers/TradingChart/api.ts

```diff
@@ -18,8 +18,10 @@ export interface CurrentKlineSubscription {
     periodString?: string;
 }
 
+const getHistoryApi = (): string => isFinexEnabled() ? finexUrl() : arkeUrl();
+
 const makeHistoryUrl = (market: string, resolution: number, from: number, to: number) =>
-    `${arkeUrl()}/public/markets/${market}/k-line?period=${resolution}&time_from=${from}&time_to=${to}`;
```

Comment:
Can you apply the following helper to from and to parameters:

const tsPeriod = (ts: number, period: number) => ts - ts%period*60;